nix/sources.json

```diff
@@ -1,14 +1,26 @@
 {
     "agent-rs": {
-        "branch": "paulliu/add-cargo-lock",
+        "branch": "main",
         "description": "A collection of libraries and tools for building software around the Internet Computer, in Rust.",
         "homepage": "https://sdk.dfinity.org/",
-        "owner": "ninegua",
+        "owner": "dfinity",
         "repo": "agent-rs",
-        "rev": "4a22e590516bc79ec3c75a320f7941e7762ea098",
-        "sha256": "0sacddc34nlfgldqghlwchgzjki177h5dsgpmdv70cm8hfy0sg7l",
+        "rev": "71987142bb94a55ac0cb74a52396e01b6a142a16",
+        "sha256": "0hv3cg3f8jl0qpzm1qfrg1x7vbcqqv0mzjrgmh4g3w6f8xfj68is",
+        "type": "tarball",
+        "url": "https://github.com/dfinity/agent-rs/archive/71987142bb94a55ac0cb74a52396e01b6a142a16.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "fenix": {
```

Comment:
I've switched `nixpkgs-mozilla` to `fenix`, since the latter works better on aarch64-darwin (e.g. it doesn't try to build rustc from scratch).